include/drivers/sensor/bq274xx.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Move this to `drivers/sensor/bq274xx/bq274xx.h`. We don't need to have register definition in include

Reply:
Well, the whole idea is to be able to check the status flags in application code, so the definitions are helpers to get the info from it.

Reply:
Well status register represents the current state of various async conditions of the sensor. IMO, this needs to be handled with trigger/interrupt.

Reply:
The trigger/interrupt could be done for `GPOUT` and `BIN` pins, but not for the status register that needs to be polled, right?

Reply:
Not with polling. GPOUT asserts the host CPU and the ISR in-turn. ISR needs to check the status register for the condition to match IMO

Reply:
This could be another addition, but would require even more configuration of the device. As you need to set the purpose of the `GPOUT` pin and link it to the application.
This PR is kept simple.